packages/eslint-plugin/src/rules/consistent-generic-constructors.ts

```diff
@@ -118,9 +118,11 @@ export default createRule<Options, MessageIds>({
           const extraComments = new Set(
             context.sourceCode.getCommentsInside(lhs.parent),
           );
-          context.sourceCode
-            .getCommentsInside(lhs.typeArguments)
-            .forEach(c => extraComments.delete(c));
+          for (const c of context.sourceCode.getCommentsInside(
+            lhs.typeArguments,
+          )) {
+            extraComments.delete(c);
+          }
```

Comment:
This is one of the cases where I strongly believe the original case is more readable and thus the rule is not useful to be enforced.

Reply:
I think I agree. And enabling this rule did not surface any type-related issues

Reply:
Yeah it's a pity, a lot of the other cases I do like more, but this one not so much... it would be nice if there'd be a way to encodify this. Maybe something around avoiding changing places with this kind of chaining?

Reply:
Yeah, I agree. Unfortunately `no-array-for-each` has no configuration options.

At work I do use `no-array-for-each`, because I feel like the upsides of better consistency and better readability _most of the time_ is better than the downside of worse readability occasionally.